apps/fluent-tester/src/RNTester/Consts.ts

```diff
@@ -0,0 +1,51 @@
+// Button Test Page
+export const HOMEPAGE_BUTTON_BUTTON = 'Homepage_Button_Button'; // Button on the homepage that navigates to the selected test page
```

Comment:
Is splitting these out part of a potential phase 2?

Reply:
Yes, I already have another branch where this is done, will be putting in a PR when this one merges.